mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java

```diff
@@ -176,14 +177,24 @@ public void inflateActivationDate() {
         tv_activationDate.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                newDatePicker.show(getActivity().getSupportFragmentManager(), FragmentConstants
-                        .DFRAG_DATE_PICKER);
+                Dialog dialogFrg=newDatePicker.getDialog();
```

Comment:
WhitespaceAround: '=' is not preceded with whitespace.

Reply:
If I add whitespace before and after the'=' will it pass?

Reply:
Yeah sure

Reply:
fulfill all the comments